File: Afb88/src/main/java/com/nanyang/app/login/LoginActivity.java
package com.nanyang.app.login;

import android.os.Bundle;
import android.view.View;

import com.nanyang.app.R;
import com.unkonw.testapp.libs.base.BaseActivity;
import com.unkonw.testapp.libs.utils.ToastUtils;
import com.unkonw.testapp.login.LoginContract;
import com.unkonw.testapp.login.LoginInfo;
import com.unkonw.testapp.login.LoginPresenter;


/**
 * Created by Administrator on 2017/1/10 0010.
 */

public class LoginActivity extends BaseActivity<LoginPresenter> implements LoginContract.View {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);
        createPresenter(new LoginPresenter(this));
    }

    @Override
    public void initView() {

    }

    @Override
    public void bindEvent() {

    }

    public void onCLickLogin(View v) {
//        presenter.login(new LoginInfo("0", "DLDLDLYY15", "111111", "Android", "1.0").getMap());
        presenter.login(new LoginInfo("0", "DLDLDLYY15", "111111", "Android", "1.0"));
    }


    @Override
    public void onGetData(final String data) {
        ToastUtils.showShort(data);
    }

}

File: app/src/main/java/gaming178/com/casinogame/load/LoginActivity.kt
package gaming178.com.casinogame.load

import android.os.Bundle
import gaming178.com.baccaratgame.R
import gaming178.com.casinogame.base.BaseActivity
import gaming178.com.mylibrary.allinone.util.AppTool

public class LoginActivity : BaseActivity() {
    override fun getLayoutRes(): Int {
        return R.layout.activity_login
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun initData(savedInstanceState: Bundle?) {
        super.initData(savedInstanceState)
        AppTool.setAppLanguage(mContext, AppTool.getAppLanguage(mContext))

    }
}